projects/fluid_simulation/sim.cpp

```diff
@@ -12,13 +12,17 @@
 #include "ppx/application.h"
 #include "ppx/config.h"
 #include "ppx/grfx/grfx_format.h"
-#include "ppx/timer.h"
 
 #include <algorithm>
 #include <cmath>
 
 namespace FluidSim {
 
+// In a normal game, animations are linked to the frame delta-time to make then run
+// as a fixed perceptible speed. For our use-case (benchmarking), determinism is important.
+// Targeting 60ips.
```

Comment:
ips? iterations per second?

Reply:
images per seconds. I'll expand the abbreviation